aait/backend/group-2B/Application/Contracts/Persistence/IInteractionRepository.cs

```diff
@@ -2,4 +2,8 @@
 
 namespace SocialSync.Application.Contracts.Persistence;
 
-public interface IInteractionRepository : IGenericRepository<Interaction> { }
+public interface IInteractionRepository : IGenericRepository<Interaction> {  
+     Task<List<Interaction>> GetAllCommentInteractions(int PostId);
```

Comment:
Prefer to use Async suffix for Async method names(for readability and consistency).

Reply:
I have resolved it.